composer.json

```diff
@@ -24,7 +24,8 @@
         "php": ">=7.1",
         "pimple/pimple": "^3.0",
         "aura/sqlquery": "^2.6",
-        "doctrine/cache": "^1.6"
+        "doctrine/cache": "^1.6",
+        "brick/geo": "^0.5.0"
```

Comment:
Are we sure we want to force users to be stuck on `0.5.x` versions ? `brick/geo` has `0.9.0` as latest version.

Reply:
Can you please take a look at https://github.com/jmikola/geojson ?